index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css" type="text/css" media="all" />
+    <title>Capstone Website</title>
+</head>
+<body class="body">
+  <section class="content">
+    <header id="top-bar">
+      <div class="so-container">
+        <div class="socials">
+            <img id="facebook" src="./images/socials/facebook.svg" alt="Facebook Handle.">
+            <img id="twitter" src="./images/socials/twitter.svg" alt="Twitter Handle.">
+            <ul class="nav-list">
+                <li id="top-1">English</li>
+                <li id="top-2">My Page</li>
+                <li id="top-3">Logout</li>
+            </ul>
+            </div>
+            </div>
+            <nav id="nav-bar">
+              <img id="ham-menu" src="./images/ham.png" alt="Hamburger Menu" />
+               <img id="logo" src="./images/nba-logos/nba-logo.svg" alt="Logo">
+               <img id="exit" src="./images/close-button.png" alt="Exit Menu" />
```

Comment:
- [ ] [OPTIONAL] I can see that you used images for different icons in the page in the header part. This takes up large space therefore being very expensive for the browser to load the page. As you understand, the reduction in the web performance will discourage seriously the user experience. Please use icons that are imported from font awesome for example because they are small in space and are more reliable 😉